app/controllers/api/questions_controller.rb

```diff
@@ -1,10 +1,11 @@
 class Api::QuestionsController < ApplicationController
-  load_and_authorize_resource
+  # load_and_authorize_resource
```

Comment:
this was causing me some problems, i think we need to redefine cancancan abilities

Reply:
things may break if I uncomment it out is that ok?

Reply:
should i just remove

Reply:
I have made this change locally for testing as well, I just want to make sure that we come back to this at some point and make it work